spec/console/board_renderer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "console/board_renderer"
+require "core/board"
+
+describe BoardRenderer do
+  it "should turn a board into renderable lines" do
+    board = Board.new
+    board_renderer = BoardRenderer.new
+
+    expected_lines = [
+      "1 | 2 | 3",
+      "4 | 5 | 6",
+      "7 | 8 | 9"
+    ].join("\n")
+
+    actual_lines = board_renderer.render(board)
```

Comment:
I would put this above the `expected_lines` declaration in order to follow 'Arrange, Act, Assert'. I think declaring what is expected is part of the test assertion because if it were shorter you'd have inlined it as part of the assertion.